ibm/config.go

```diff
@@ -594,91 +594,368 @@ func (sess clientSession) IAMNamespaceAPI() (*ns.IbmCloudFunctionsNamespaceAPIV1
 
 // CIS Zones Service
 func (sess clientSession) CisZonesV1ClientSession() (*ciszonesv1.ZonesV1, error) {
+	if sess.cisZonesErr == nil {
+		cisZonesV1Opt := &ciszonesv1.ZonesV1Options{
```

Comment:
I am pretty sur you can create a factory for this pattern for:

*  ciszonesv1.ZonesV1Options
* cisdnsrecordsv1.DnsRecordsV1Options
* cisglbv1.GlobalLoadBalancerV1Options
* ...

Reply:
will try that later.